src/components/ContentNode.vue

```diff
@@ -226,14 +238,12 @@ function renderNode(createElement, references) {
       ...rest,
       metadata,
     };
-    const figureContent = [renderChildren([node])];
-    if ((title && abstract.length) || abstract.length) {
```

Comment:
In case there is no `title` the `or` conditional makes it valid because there is `abstract.length` anyways.
I didn't understand why the `title` condicional was included in the first place. 🤔